tests/test_store/test_mmap.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import mmap
+from typing import TYPE_CHECKING
+
+import pytest
+
+import zarr
+from zarr.core.buffer import Buffer, cpu
+from zarr.storage import LocalStore
+from zarr.testing.store import StoreTests
+
+if TYPE_CHECKING:
+    import pathlib
+
+
+class MemoryMappedDirectoryStore(LocalStore):
+    def _fromfile(self, fn: str) -> memoryview:
+        with open(fn, "rb") as fh:
+            return memoryview(mmap.mmap(fh.fileno(), 0, prot=mmap.PROT_READ))
```

Comment:
I don't think `_fromfile` will ever be invoked, since it's not part of the `LocalStore` API